src/Achieve.Aspire.AzureProvisioning/Bicep/Internal/BicepOutput.cs

```diff
@@ -27,12 +27,15 @@ private PropertyAccessSyntax GetOutputPathSyntax()
         // This generates depending on the output path, super hacky for now
         // There will be a better way, but for now...
         var splitPath = Path.Split('.');
+        var first = splitPath[0];
+        var second = splitPath.Length > 1 ? splitPath[1] : splitPath[0];
+        
         var propertyAccess =
             new PropertyAccessSyntax(
-                new VariableAccessSyntax(SyntaxFactory.CreateIdentifier(splitPath[0])),
+                new VariableAccessSyntax(SyntaxFactory.CreateIdentifier(first)),
                 SyntaxFactory.DotToken,
                 null,
-                SyntaxFactory.CreateIdentifier(splitPath[1]));
+                SyntaxFactory.CreateIdentifier(second));
```

Comment:
Approving this - but as a note for the future this should probably throw an error? It's not a case that it will ever hit (hopefully) as a property access requires at least 2 parts, but just to be safe.

Reply:
Yeah, there's definitely a better way to do this, but I just wanted to get something out there that worked for my new component while also not breaking what was there. There's a lot of opportunity for some helper methods like this in the plumbing as I get more familiar with it all.

Reply:
Agree - I haven't posted about this library much at all yet until there's more resources in here, though I do think it's super valuable for a) support of more resources that Aspire probably won't have support for and b) configuring things "properly" for a production environment.

If it gets more traction, will definitely clean things up more 🙂